src/pages/course-page/CoursePage.tsx

```diff
@@ -73,44 +76,23 @@ const CoursePage = () => {
       <p className={styles.title}>{lessons?.title}</p>
       <div className={styles.videoList}>
         <div className={styles.video}>
-          <h3 className={styles.subTitle}>
-            Lesson: {sortedLessons?.[currentLesson]?.title}
-          </h3>
+          <h2 className={styles.subTitle}>Lesson: {title}</h2>
           <video
             className={styles.myVideo}
             controls
             poster={poster}
             ref={videoRef}
           />
-          <div className={styles.courseInfo}>
-            <div>
-              <h6>*To speed up the video, press {VideoPlayerKeys.SPEED_UP}</h6>
-              <h6>
-                *To slow down the video, press {VideoPlayerKeys.SLOW_DOWN}
-              </h6>
-            </div>
-            <div>
-              <h6 className={styles.date}>
-                <CalendarTodayIcon />
-                Launch date:
-                {lessons?.launchDate
-                  ? new Date(lessons?.launchDate).toLocaleDateString('en-GB')
-                  : 'No date available'}
-              </h6>
-            </div>
-          </div>
+          <CourseInfo lessons={lessons} />
         </div>
         <div className={styles.lessons}>
-          <h4 className={styles.content}>Lessons:</h4>
+          <h2 className={styles.content}>Lessons:</h2>
           {sortedLessons?.map(lesson => (
             <ListLessons
-              duration={lesson.duration}
-              setCurrentLesson={setCurrentLesson}
-              order={lesson.order}
               key={lesson.id}
-              title={lesson.title}
+              lesson={lesson}
```

Comment:
Why did you decide to transfer not only the necessary data (duration, order, title), but the entire lesson?

Reply:
> Why did you decide to transfer not only the necessary data (duration, order, title), but the entire lesson?

I decided to put all this things that came from lesson of Course Page in LessonList as it is and then destructuring all properties I need in component. + It is easier to read code in Course Page from the point of semantic.